lang/solidity/Portfile

```diff
@@ -4,10 +4,9 @@ PortSystem          1.0
 
 PortGroup           github 1.0
 PortGroup           cmake 1.1
-PortGroup           boost 1.0
 
-github.setup        ethereum solidity 0.8.19 v
-github.tarball_from downloads
+github.setup        ethereum solidity 0.8.24 v
+github.tarball_from releases
 distname            ${github.project}_${github.version}
```

Comment:
do you need to set the "distname" - typically the "github" PortGroup takes care of these things.

Reply:
The default is `${github.project}-${github.version}.

ref: https://guide.macports.org/#reference.portgroup.github